src/css/parse.js

```diff
@@ -828,30 +903,36 @@ function getBrackets() {
  * @return {Number} Length of the class selector
  */
 function checkClass(i) {
-  var l;
+  const start = i;
+  let l;
 
   if (i >= tokensLength) return 0;
 
-  if (tokens[i].class_l) return tokens[i].class_l;
+  if (tokens[i++].type !== TokenType.FullStop) return 0;
```

Comment:
Since you seem to prefer "skip" comments over such code, I would propose:
```
if (tokens[i].type === TokenType.FullStop) i++;
else return 0;
```
That would be consistent with other places.